File: Services/DB_Handler/models/URLStorage.py
from sqlalchemy import Column, Integer, String, DateTime, Text
from sqlalchemy.orm import relationship
from .base import Base


class URLStorage(Base):
    __tablename__ = 'url_storage'

    url_id = Column(Integer, primary_key=True, nullable=False)
    url = Column(String, nullable=False)
    description = Column(Text)
    search_date = Column(DateTime)

File: Services/Public/Public.Domain/UrlStorage.cs
using System.ComponentModel.DataAnnotations;

namespace Public.Domain;

public class UrlStorage
{
    [Key] public int url_id { get; set; }
    
    [Required] public string url { get; set; }

    [Required] public string description { get; set; }

    [Required] public DateTime search_date { get; set; }
}